theano/printing.py

```diff
@@ -558,7 +558,8 @@ def pydotprint(fct, outfile=None,
                assert_nb_all_strings=-1,
                return_image=False,
                ):
-    """Print to a file (png format) the graph of a compiled theano function's ops.
+    """Print to a file (png or svg format) the graph of a compiled theano
```

Comment:
In fact, we support all format that pydot support. What about "(all format pydot support: e.g. png and svg)"?

thanks